components/automate-ui/src/app/pages/automate-settings/automate-settings.component.ts

```diff
@@ -33,77 +34,7 @@ import { TelemetryService } from '../../services/telemetry/telemetry.service';
 
 export class AutomateSettingsComponent implements OnInit, OnDestroy {
 
-  private defaultFormData: DefaultFormData = {
```

Comment:
Changed this to a function to allow modifying the variables based on `isDesktopView`.